app/assets/stylesheets/earthworks.css

```diff
@@ -152,3 +152,37 @@ Uncomment lines 142-144
     display: none;
   }
 }
+
+.filter .btn-outline-secondary {
+  --bs-btn-border-color: var(--stanford-fog-light);
+  --bs-btn-color: var(--stanford-black);
+  background-color: var(--stanford-fog-light);
+  --bs-btn-bg-color: var(--stanford-fog-light);
+  --bs-btn-hover-color: var(--stanford-black);
+  --bs-btn-hover-border-color: var(--stanford-fog-light);
+  --bs-btn-active-border-color: var(--stanford-fog-light);
+  --bs-btn-active-color: var(--stanford-black);
+  --bs-btn-padding-y: .25rem;
+}
+
+.applied-filter .remove:hover {
+  --bs-btn-hover-border-color: var(--stanford-fog-light);
+  --bs-btn-hover-bg: var(--stanford-fog-light);
+  --bs-btn-hover-color: var(--stanford-digital-red);
+  text-decoration: none;
```

Comment:
Can you use `--bs-link-hover-decoration` instead?

Reply:
If you use the variables, you don't need the `:hover` pseudo-selector.

Reply:
Nope. Unfortunately geoblacklight css has the line 

a {
  color: rgba(var(--bs-link-color-rgb), var(--bs-link-opacity, 1));
  text-decoration: underline;
}

Reply:
Can we change that in Geoblacklight? Is seems like it's an impediment for us, so it will likely cause others trouble too.  When you run into something like this, I find it helpful if you write in a comment why you are doing it.

Reply:
Can you show me where you see that? I'm not seeing it on https://earthworks-uat.stanford.edu/?f%5Bdct_publisher_sm%5D%5B%5D=Collins+Bartholomew+Ltd.&q=&search_field=all_fields
<img width="622" alt="Screenshot 2024-08-20 at 4 48 05 PM" src="https://github.com/user-attachments/assets/c4c4f9e7-d5e0-475d-841c-0e8bb3bcdf36">

Reply:
Are you sure it's not https://github.com/sul-dlss/component-library/blame/main/styles/button.css#L2 that you're hitting.  I'm pretty sure DLUX said that the hover state of buttons should have underline.

Reply:
Nope. Element above.

Reply:
@jcoyne 
<img width="640" alt="Screenshot 2024-08-20 at 5 59 26 PM" src="https://github.com/user-attachments/assets/39865733-ac0b-4b65-addc-808f881d31e2">

Reply:
Let's work with the designers to understand what they want for the component library, rather than having to override it in every project where we use the component library.

Reply:
Should I remove the line for now?

Reply:
Sounds good.